src/CreditCard/CreditCardForm.js

```diff
@@ -38,15 +40,13 @@ const CreditCardForm = ({ card, onchange, handleChangeCode, handleChangeDate, ha
     e.returnValue = false; //IE
   };
 
-  const ValidationForm = (e) =>{
-
-    e.preventDefault()
```

Comment:
Why removing this?

Reply:
No fue eliminada, se creo una nueva funcion para validad errores del formulario